server/utils/utils.ts

```diff
@@ -37,6 +48,47 @@ export const formatDateOrDefault = (placeHolder: string, dateFormat: string, dat
   return format(parseISO(date), dateFormat)
 }
 
+const getDateSelection = (date: Date, selectedDate: Date, amount = 12): ProcessedDateSelection[] => {
```

Comment:
One for when this gets picked back up - probably worth putting these date utils in their own file i.e. `utils/date.ts` and renaming this file something like `utils/common.ts` and updating with test files accordingly i.e. `utils/common.test.ts`, `utils/date.test.ts` etc. Worth seeing what other ones could be split into their own files too, for example there's a few in this file that could be moved to `utils/string.ts`.